### ISSUE TITLE
Replace CurationConcerns::NullLogger with ActiveFedora::NullLogger

### ISSUE THREAD

**Issue by awead** [projecthydra/curation_concerns#1007]
There's a PR in on ActiveFedora to use NullLogger, https://github.com/projecthydra/active_fedora/pull/1153, which is the same as Curation Concerns' null logging object. Once that PR is merged, we can just use that one instead.

**Comment by botimer** [comment]
The AF PR is merged, but there is no release containing it yet. Also, it is not clear when it is safe to bump the gemspec dependency to 11.0.x. It's currently '>= 10.3.0.rc1'.

I'm happy to tidy this up with some advice on the versioning.

**Comment by awead** [comment]
I think all we need to do is cut a new release of AF.

**Comment by jcoyne** [comment]
If we require AF 11, you'll transitively require rdf 2, which may be a problem for some users

**Comment by awead** [comment]
I can backport the NullLogger to version 10, if needed.

**Comment by botimer** [comment]
Got it. Since the point is that the logger does nothing, I'd be inclined to leave it in place and close this. Not too much value in binding across gems here when there's no real risk of desync or duplication. And there's a downside in dangling issues.

**Comment by awead** [comment]
The reason I filed the ticket was so that we could remove the CurationConcerns::NullLogger code from curation concerns because it's duplicating was ActiveFedora is already doing... so lighten the overall codebase.

**Comment by awead** [comment]
meh. I'm inclined to backport because it'd be nice to not have the duplication, alternatively, we could tag this for a version of CC that requires AF 11.

**Comment by botimer** [comment]
Right. I'm not really invested in any of these approaches -- it's just not the obvious housekeeping thing it seemed to be. A release of AF is needed to address it or a decision made that it's okay as-is.

I'm happy to port/PR to AF 10.3-stable and bump the CC gemspec if that's the winner. Up to you two.
